src/main/java/com/theocean/fundering/domain/comment/dto/CommentResponse.java

```diff
@@ -10,12 +10,13 @@ public class CommentResponse {
   @Getter
   public static class findAllDTO {
```

Comment:
멘토님이 피드백 주신 것처럼 static class 이름을 대문자로 시작하면 좋을 꺼 같아요!
```suggestion
  public static class FindAllDTO {
```